core/domain/activity_jobs_one_off_test.py

```diff
@@ -0,0 +1,98 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Tests for Activity-related jobs."""
+
+from core.domain import exp_domain
+from core.domain import collection_domain
+from core.domain import activity_jobs_one_off
+from core.domain import exp_services
+from core.domain import collection_services
+from core.domain import rights_manager
+from core.domain import search_services
+from core.domain import user_services
+from core.platform import models
+from core.platform.taskqueue import gae_taskqueue_services as taskqueue_services
+from core.tests import test_utils
+
+gae_search_services = models.Registry.import_search_services()
+
+class OneOffReindexActivitiesJobTest(test_utils.GenericTestBase):
+
+    EXP_ID = 'exp_id'
+    COLLECTION_ID = 'collection_id'
+
+    def setUp(self):
+        super(OneOffReindexActivitiesJobTest, self).setUp()
+
+        self.signup(self.OWNER_EMAIL, self.OWNER_USERNAME)
+        self.owner_id = self.get_user_id_from_email(self.OWNER_EMAIL)
+        self.owner_a = user_services.UserActionsInfo(self.owner_id)
+
+        # Setup explorations
+        explorations = [exp_domain.Exploration.create_default_exploration(
+            '%s' % i,
+            title='title %d' % i,
+            category='category%d' % i
+        ) for i in xrange(3)]
+
+        for exp in explorations:
+            exp_services.save_new_exploration(self.owner_id, exp)
+            rights_manager.publish_exploration(self.owner_a, exp.id)
+
+        # Setup collections
+        collections = [collection_domain.Collection.create_default_collection(
+            '%s' % i,
+            title='title %d' %i,
+            category='category%d' %i
+        ) for i in range(3, 6)]
+
+        for collection in collections:
+            collection_services.save_new_collection(self.owner_id, collection)
+            rights_manager.publish_collection(self.owner_a, collection.id)
+
+        self.process_and_flush_pending_tasks()
+
+
```

Comment:
Remove one of these newlines; we separate methods of a class by one newline. Only top-level function/class definitions are separated by two newlines.